internal/datanode/segment_replica.go

```diff
@@ -176,6 +199,10 @@ func (replica *SegmentReplica) addNewSegment(segID, collID, partitionID UniqueID
 		checkPoint: segmentCheckPoint{0, *startPos},
 		startPos:   startPos,
 		endPos:     endPos,
+
+		pkFilter: bloom.NewWithEstimates(100000, 0.005), // TODO silverxia change to unified configuration
```

Comment:
Should not use a literal directly, move to a unified place.

Reply:
Fixed in new push